DOCUMENTATION.md

```diff
@@ -1,810 +1,807 @@
-    address_line_1: 'Her Majesty The Queen',  # required
-    address_line_2: 'Buckingham Palace', # required
-    address_line_3: 'London',
-    postcode: 'SW1 1AA',  # required
+template_id: "f33517ff-2a88-4f6e-b855-c550268ce08a",
+```
 
-    ... # any other personalisation found in your template
-  },
-  reference: "your_reference_string"
-) # => Notifications::Client::ResponseNotification
+#### personalisation (required)
+
+The personalisation argument always contains the following parameters for the letter recipient's address:
+
+- `address_line_1`
+- `address_line_2`
+- `postcode`
+
+Any other placeholder fields included in the letter template also count as required parameters. You must provide their values in a hash. For example:
+
+
+```ruby
+personalisation: {
+  address_line_1: 'The Occupier',  # mandatory address field
+  address_line_2: '123 High Street', # mandatory address field
+  postcode: 'SW14 6BH',  # mandatory address field
+  name: 'John Smith', # field from template
+  application_date: '2018-01-01' # field from template
+},
 ```
 
-</details>
+#### personalisation (optional)
 
+The following parameters in the letter recipient's address are optional:
 
-#### Response
+```ruby
+personalisation: {
+  address_line_3: 'Richmond',  # mandatory address field
+  address_line_4: 'London', # mandatory address field
+  address_line_5: 'Middlesex', # mandatory address field
+  address_line_6: 'UK', # mandatory address field
+},
+```
 
-If the request is successful, a `Notifications::Client:ResponseNotification` is returned.
+#### reference (optional)
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+A unique identifier. This reference identifies a single unique notification or a batch of notifications. If you do not have a reference, you must pass in an empty string or `null`.
 
 ```ruby
-letter => Notifications::Client::ResponseNotification
+reference: 'your_reference_string';
+```
 
-letter.id           # => uuid for the notification
-letter.reference    # => Reference string you supplied in the request
-letter.content      # => Hash containing body => the body of the letter sent to the recipient, with placeholders replaced
-                    #                    subject => the main heading of the letter
-letter.template     # => Hash containing id => id of the template
-                    #                    version => version of the template
-                    #                    uri => url of the template
-letter.uri          # => URL of the notification
+### Response
+
+If the request to the client is successful, the client will return a `Notifications::Client:ResponseNotification`:
+
+```ruby
+@content={"notification content"},
+@id="8a6f5cb9-98d1-436d-bdb0-15cc2edc8e65",
+@reference=nil,
+@template=
+ {"id"=>"f33517ff-2a88-4f6e-b855-c550268ce08a",
+  "uri"=>"https://api.notify.works/services/d33f763f-d8b6-4591-b272-ec6127572e5a/templates/2dd69cce-c0d2-470f-a6c1-8900a5368e04",
```

Comment:
The start of this should be `https://api.notifications.service.gov.uk/services...`